Portfolio/index.html

```diff
@@ -38,7 +38,7 @@
     <div class="i">
       GET TO KNOW ME
       <p style="color: #9ABEE9;font-family: Roboto;font-size:20px;font-style: normal;letter-spacing: 1.05px;">I'm a seasoned front-end web developer with a passion for crafting exceptional<br> digital experiences. With over 2 years of hands-on experience, i bring a deep<br> proficiency in HTML, CSS, JavaScript, TypeScript, and a keen mastery of<br> modern frameworks like React.js, React Native, Svelte.js.<br>

```

Comment:
There are multiple non-breaking spaces (ASCII code 160) between 'tackle' and 'any'. Please remove these non-breaking spaces and use regular spaces between words.